_data/config-metadata.csv

```diff
@@ -1,20 +1,20 @@
 field,display_name,browse_link,external_link,dc_map,schema_map
 objectid,Identifikator,,,DCTERMS.identifier,identifier
 title,Titel,,,DCTERMS.title,headline
-creator,Ersteller*in,,,DCTERMS.creator,creator
-date,Datum,,,DCTERMS.date,dateCreated
-era,Epoche,true,,DCTERMS.temporal,
 description,Beschreibung,,,DCTERMS.description,description
 subject,Thema,true,,DCTERMS.subject,keywords
-publisher,Rechteinhaber*in,true,,DCTERMS.publisher,publisher
-source,Quelle,,,DCTERMS.source,
-relation,Link,,true,DCTERMS.relation,
-hasVersion,IIIF Manifest,,true,DCTERMS.hasVersion,version
-type,Typ,,,DCTERMS.type,
+era,Epoche,true,,DCTERMS.temporal,
+isPartOf,Band,,true,DCTERMS.isPartOf,isPartOf
+creator,Ersteller*in,,true,DCTERMS.creator,creator
+publisher,Rechteinhaber*in,,true,DCTERMS.publisher,publisher
```

Comment:
"Rechteinhaber" and "Publisher" are not the same. I'd go for "Verantwortliche Gedächtnisinstitution" or "Aufbewahrungsort".